index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/b08094b2a2.js" crossorigin="anonymous"></script>
+    <title>Personal Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav><i class="fas fa-bars"></i></nav>
+    </header>
+    <section>
+        <h1 class="header">Hey there.
+            <br>I’m Ian.
+           <hx class="text_color">I’m a software developer</hx>
```

Comment:
- [ ] `hx` is not a valid tag, it could be better to use a number that represents the hierarchy of the content In this case please 🙏🏽 replace the x with a number e.g. `h2` 

> As requested by a previous reviewer it is recommended to use an hx tag for your subtitle instead of a font tag, for better accessibility and semantics.